CSI_Greenville_HIST8510_Final_Project-AR.qmd

```diff
@@ -0,0 +1,390 @@
+---
+title: "CSI:Dixie Greenville"
+author: "Candy Boatwright"
+format: html
+---
+
+```{r}
```

Comment:
If you add #| echo: false to the top of this block it will prevent it from showing up in the final rendered version